content/docs/command-reference/metrics/index.md

```diff
@@ -63,10 +33,8 @@ stages:
           cache: false
 ```
 
-> `cache: false` above specifies that `summary.json` is not tracked or
-> <abbr>cached</abbr> by DVC (`-M` option of `dvc run`). These metrics files are
-> normally committed with Git instead. See `dvc.yaml` for more information on
-> the file format above.
+Note that metrics files are normally committed with Git (that's what
+`cache: false` above is for). See `dvc.yaml` for more information.
 
 ### Supported file formats
 
```

Comment:
This should be moved to the concepts page.

Cmd ref indices are all going to end up being pretty small after this change. Maybe even remove them all (but for now this one can stay).

Reply:
> This should be moved to the concepts page.

OK, done.

> Maybe even remove them all.

Yes, would also help with SEO on two fronts: 1. removing duplicate/similar content, 2. expanded/improved content on remaining pages

Reply:
Looks good, thanks. And yes, I'll prob remove the indices as we work on each concept in it's own PR.